prototype2/multigrid/mgmesytec/Hit.h

```diff
@@ -1,6 +1,31 @@
-/// Copyright (C) 2017-2018 European Spallation Source, ERIC. See LICENSE file
+/* Copyright (C) 2017-2018 European Spallation Source, ERIC. See LICENSE file */
 //===----------------------------------------------------------------------===//
 ///
+///                              WARNING
+///
+///                              ACHTUNG
+///
+///                              AVISO
+///
+///                              ADVARSEL
+///
+///                              DĖMESIO
+///
+///                              UWAGA
+///
+///
+///
+///          MODIFY THIS FILE ONLY AFTER UNDERSTANDING THE FOLLOWING
+///
+///
+///   Any changes to non-static variable definitions will likely break h5 file
+/// writing and compatibility. If you rename, reorder or change the type of any
+/// of the member variables, you MUST:
+///    A) Increment FormatVersion by 1
+///    B) Ensure the hdf5 TypeTrait maps the struct correctly
+///
+/// If you cannot ensure the above, consult someone who can.
+///
```

Comment:
I don't like the terminology Hit - when to use Hit and when to use readout?

If the readout system doesn't produce readouts, then what?

Reply:
Hits are what the detector people apparently call it. I just went with Anton's terminology. It kind of makes sense -- a few hits in coincidence constitutes an event. No?

Reply:
Well, I sort of disagree. Hit indicates a physical event to me, whereas readout just 
is what it is: the electronics reading adc values out. Sometimes these were caused by
a hit of some sort (neutron, electron, gamma), but other times they are just readout 
because of system configuration (vmm3 neighbour, or read-all-channels). Anyway we now have two different words for similar stuff.  Not important now, but maybe for discussion later?